specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/myworkbooks_API.json

```diff
@@ -0,0 +1,554 @@
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "MyWorkbooksListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/MyWorkbook"
+          },
+          "description": "An array of private workbooks."
+        }
+      },
+      "description": "Workbook list result."
+    },
+    "MyWorkbook": {
+      "description": "An Application Insights private workbook definition.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MyWorkbookResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "The kind of workbook. Choices are user and shared.",
+          "enum": [
+            "user",
+            "shared"
+          ],
+          "x-ms-enum": {
+            "name": "SharedTypeKind",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook for an Azure resource.",
+          "$ref": "#/definitions/MyWorkbookProperties"
+        }
+      }
+    },
+    "MyWorkbookProperties": {
+      "description": "Properties that contain a private workbook.",
+      "required": [
+        "displayName",
+        "kind",
+        "category",
+        "serializedData",
+        "userId"
+      ],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The user-defined name of the private workbook."
+        },
+        "serializedData": {
+          "type": "string",
+          "description": "Configuration of this particular private workbook. Configuration data is a string containing valid JSON"
+        },
+        "version": {
+          "type": "string",
+          "description": "This instance's version of the data model. This can change as new features are added that can be marked private workbook."
+        },
+        "kind": {
+          "description": "Enum indicating if this workbook definition is owned by a specific user or is shared between all users with access to the Application Insights component.",
+          "x-ms-client-name": "SharedTypeKind",
+          "default": "shared",
+          "type": "string",
+          "enum": [
+            "shared",
+            "user"
+          ],
+          "x-ms-enum": {
+            "name": "SharedTypeKind",
+            "modelAsString": true
+          }
+        },
+        "timeModified": {
+          "type": "string",
+          "description": "Date and time in UTC of the last modification that was made to this private workbook definition.",
+          "readOnly": true
+        },
+        "category": {
+          "type": "string",
+          "description": "Workbook category, as defined by the user at creation time."
+        },
+        "tags": {
```

Comment:
It's not same with the `tag` in other resource. It's just a list of string.

Reply:
this inner tags inside properties is a legacy thing that existed before this resource type existed (when they were a proxy resource inside application insights, so it didn't support tags at the outer level).  in this case, it was just a list of tags, not a key:value pair like azure tags are.  (personally, we've talked about internally removing this internal tags entirely)